src/RadixDlt.NetworkGateway.PostgresIntegration/Models/LedgerTransaction.cs

```diff
@@ -179,6 +184,48 @@ internal class TransactionReceipt
 
     [Column("receipt_event_sbor_type_kinds")]
     public SborTypeKind[] EventSborTypeKinds { get; set; }
+
+    public List<TransactionReceiptEventLookup> GetEventLookups()
+    {
+        var result = new List<TransactionReceiptEventLookup>();
+
+        // TODO drop this if statement once we deploy with complete db-wipe
+        if (EventSchemaEntityIds.Length != EventSbors.Length)
+        {
+            throw new NotSupportedException("Missing EventSchemaEntityIds - wipe your database and ingest all the data");
+        }
+
+        for (var i = 0; i < EventSbors.Length; ++i)
+        {
+            result.Add(new TransactionReceiptEventLookup(EventSchemaEntityIds[i], EventSchemaHashes[i]));
+        }
+
+        return result;
+    }
+
+    public List<TransactionReceiptEventData> GetEvents()
```

Comment:
we already had similar extension TransactionReceiptExtensions with type TransactionReceiptEventData which is probably unused now.